content/docs/start/data-and-model-versioning.md

```diff
@@ -247,6 +247,16 @@ defines data file versions. Git itself provides the version control. DVC in turn
 creates these `.dvc` files, updates them, and synchronizes DVC-tracked data in
 the <abbr>workspace</abbr> efficiently to match them.
 
+## Model versioning
+
+DVC helps you to handle model files as well. Models in a project usually change
+more frequently than data files and they need to be kept in sync with changes in
+other elements of a project. Model files are no different than data files when
+it comes to tracking their versions. DVC also provides means to track minor
+changes in model files without fully checking in to Git. In later sections of
+this series, you'll see how DVC enables to track changes to synchronize multiple
+model and data files.
```

Comment:
> Continuing https://github.com/iterative/dvc.org/pull/2214#pullrequestreview-593674216

I'm still not convinced we need this new section. We already say "data and models" in every section (except in **Retrieving** — let's fix that though), so if the gist here is that models are also tracked as any file normally, I think that's already implied in every other section.

Also, it can probably be summarized a bit (see feedback below) and then it's too short for a whole section anyway (could be moved to right before **Storing and sharing** if anything).

- "usually change more frequently than data files" contradicts "are no different than data files" (at first sight)
- "provides means to track minor changes in model files" - vague, what do we mean? run-cache, parameters etc. are not specifically for models.
- The last sentence seems unnecessary.